enterprise/templates/enterprise/admin/manage_learners.html

```diff
@@ -138,19 +141,41 @@
         </td>
         <td>{{ learner.created }}</td>
         <td><input type="checkbox" class="enroll-checkbox"></td>
+        <td><input type="checkbox" class="unlink-checkbox"></td>
         <td class="delete-cell"><input type="button" value="Unlink"></td>
       </tr>
       {% endfor %}
       </tbody>
     </table>
+    <h1>Deactivated learners</h1>
+    <table class="learners-table deactivated-learners">
+      <thead>
+      <tr>
+        <th>{% trans "User Email" %}</th>
+        <th>{% trans "Username" %}</th>
+      </tr>
+      </thead>
+      <tbody>
+      {% for learner in unlinked_learners %}
+      <tr class="form-row {% cycle 'row1' 'row2' %}" data-email="{{ learner.user_email }}">
+        <td class="email-container">
+          {{ learner.user_email }}
+        </td>
+        <td>{{ learner.user.username }}</td>
+      </tr>
+      {% endfor %}
+      </tbody>
+    </table>
   </div>
   <div class="forms-panel">
-    <h1>{% trans "Link learners" %}</h1>
     <form action="" method="post" enctype="multipart/form-data" id="learner-management-form">
+      <h1>{% trans "Link learners" %}</h1>
       {% csrf_token %}
       {# as_p will render the form fields wrapped in <p> tags: #}
       {{ manage_learners_form.as_p }}
       <input type="submit" value="Submit"/>
+      <h1>{% trans "Unlink learners" %}</h1>
```

Comment:
The UI could use some work. Function before form and all that, but it should be more clearly separated from the rest of the form.